src/server/services/procedures/FBICrimeData/FBICrimeData.js

```diff
@@ -315,10 +329,10 @@ Crime.regionArrestCount = async function (regionName, offense, category, startYe
 /**
  * Gets the number of arrests(for a particular offense) for the state in a certain time period
  * @param {String} stateAbbr location of the state the crime occurred
- * @param {Enum<violent_crime,propery_crime,drug,human_trafficking,gambling,juvenile,society,all>} category variable affecting crimes including examples: count, weapons, etc.
+ * @param {Enum<male,female,offense,race,monthly>} category variable affecting crimes including examples: count, weapons, etc.
```

Comment:
Was this one using the wrong values? Also the description still refers to `count`, `weapons`, etc that aren't valid options here.